bigbluebutton/greenlight
# Fixed voiceBridge (5-digit number) for phone call

## Description
If voiceBridge number is not specified then BigBlueButton will assign a random unused number for the meeting. Hence, voiceBridge number is changing for each meeting even the room is not changed. Each time new voiceBridge number should be sent to participants. This pull request allows keeping voiceBridge number the same for all meetings of the room. A random unused number is set for the room and it is the same for all the meetings.

By default, the feature is disabled. To enable the feature, "phone-call" should be added to "ROOM_FEATURES" in the .env file.

## Testing Steps

1. Enable phone-call feature on BigBlueButton to allow users to join the meeting via phone call.
1. Add "phone-call" to "ROOM_FEATURES" parameter in the .env file to enable the feature.
2. Open room creation modal, then you will see auto-assigned voiceBridge number (phone-call code)
3. Start the room
4. On the left side, you will see the phone number and fixed voiceBridge number (phone-call code).

## Screenshots (if appropriate):

![phone-call](https://user-images.githubusercontent.com/18422143/122345156-d4844d00-cf47-11eb-8323-d7b559fc543b.png)
